Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM openjdk:8
+
+# add a non-privileged user for running the application
+RUN groupadd --gid 10001 app && \
+    useradd -g app --uid 10001 --shell /usr/sbin/nologin --create-home --home-dir /app app
+
+WORKDIR /app
+
+# Install python
+RUN apt-get update && \
+    apt-get -y --no-install-recommends install python2.7 python-pip python-setuptools
```

Comment:
Is this project only python 2 compatible?

Reply:
Yup, it sure is :/